ClientSideValidator/ClientSideValidatorNaot.csproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <UsingNativeAOT>true</UsingNativeAOT>
+    <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+    <PublishDir>$(MSBuildThisFileDirectory)/bin/$(Configuration)/AppBundle/_framework</PublishDir>
+    <RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-experimental/nuget/v3/index.json</RestoreAdditionalProjectSources>
```

Comment:
Prefer a nuget.config in the root of the repo to add this feed.

Reply:
Feel free to take over the PR